chart/templates/NOTES.txt

```diff
@@ -30,10 +33,27 @@ Flower dashboard:      http{{ if .Values.ingress.flower.tls.enabled }}s{{ end }}
 {{- else }}
 You can now access your dashboard(s) by executing the following command(s) and visiting the corresponding port at localhost in your browser:
 
-Airflow dashboard:        kubectl port-forward svc/{{ .Release.Name }}-webserver {{ .Values.ports.airflowUI }}:{{ .Values.ports.airflowUI }} --namespace {{ .Release.Namespace }}
+Airflow Webserver:      kubectl port-forward svc/{{ .Release.Name }}-webserver {{ .Values.ports.airflowUI }}:{{ .Values.ports.airflowUI }} --namespace {{ .Release.Namespace }}
+
 {{- if .Values.flower.enabled }}
 {{- if or (eq .Values.executor "CeleryExecutor") (eq .Values.executor "CeleryKubernetesExecutor")}}
 Flower dashboard:         kubectl port-forward svc/{{ .Release.Name }}-flower {{ .Values.ports.flowerUI }}:{{ .Values.ports.flowerUI }} --namespace {{ .Release.Namespace }}
 {{- end }}
 {{- end }}
 {{- end }}
+
+
+{{- if .Values.webserver.defaultUser.enabled}}
+Default Webserver (Airflow UI) Login credentials:
+  username: {{ .Values.webserver.defaultUser.username }}
+  password: {{ .Values.webserver.defaultUser.password }}
```

Comment:
```suggestion
    username: {{ .Values.webserver.defaultUser.username }}
    password: {{ .Values.webserver.defaultUser.password }}
```
Just to make the spaces more consistent with Line 55

![image](https://user-images.githubusercontent.com/8811558/118132649-d498a680-b3f7-11eb-998d-09ae6941b0d9.png)